pkg/virt-launcher/notify-client/client.go

```diff
@@ -195,18 +206,17 @@ func newWatchEventError(err error) watch.Event {
 	return watch.Event{Type: watch.Error, Object: &metav1.Status{Status: metav1.StatusFailure, Message: err.Error()}}
 }
 
-func eventCallback(c cli.Connection, domain *api.Domain, libvirtEvent libvirtEvent, client *Notifier, events chan watch.Event,
-	interfaceStatus []api.InterfaceStatus, osInfo *api.GuestOSInfo, domainName string) {
-	d, err := c.LookupDomainByName(util.DomainFromNamespaceName(domain.ObjectMeta.Namespace, domain.ObjectMeta.Name))
+func eventCallback(data *domainEventData) {
```

Comment:
Why do you use a pointer instead of a regular value like it was before?

Reply:
in order to save data copy, i can use "by value" if needed